pkg/apis/kops/validation/validation.go

```diff
@@ -922,6 +922,10 @@ func validateKubelet(k *kops.KubeletConfigSpec, c *kops.Cluster, kubeletPath *fi
 		if k.MemorySwapBehavior != "" {
 			allErrs = append(allErrs, IsValidValue(kubeletPath.Child("memorySwapBehavior"), &k.MemorySwapBehavior, []string{"LimitedSwap", "UnlimitedSwap"})...)
 		}
+
+		if k.SeccompDefault != nil && c.IsKubernetesLT("1.22") {
+			allErrs = append(allErrs, field.Forbidden(kubeletPath.Child("seccompDefault"), "seccompDefault is only available in Kubernetes >=1.22"))
+		}
```

Comment:
Please remove the validation. K8s 1.22 is no longer supported for some time now.